### PR TITLE
SCMInput enablement proposed api

### PR DESCRIPTION
**Pull request by lszomoru**

### CODE REVIEW

**Review by joaomoreno — DISMISSED** [review]

File: src/vs/workbench/api/common/extHostSCM.ts [hunk 1]
```diff
@@ -249,6 +249,23 @@ export class ExtHostSCMInputBox implements vscode.SourceControlInputBox {
 		this.#proxy.$setValidationProviderIsEnabled(this._sourceControlHandle, !!fn);
 	}
 
+	private _enabled: boolean = true;
+
+	get enabled(): boolean {
+		return this._enabled;
```
Comment (jrieken) [inline src/vs/workbench/api/common/extHostSCM.ts:255]: Same, enforce API proposal

**Review by jrieken — CHANGES_REQUESTED** [review]
👮 Please check that extensions are allowed to use the proposal, also update allow-list git then

File: src/vs/workbench/api/common/extHostSCM.ts (hunk 1 above)
Comment (lszomoru) [reply src/vs/workbench/api/common/extHostSCM.ts:256]: Fixed

**Review by jrieken — APPROVED** [review]
Thanks

**Comment by lszomoru** [comment]
I have validated that the API works from an extension (ex: `git`) though the adoption of the API will be in a separate PR.

**Pull request merged**